packages/ui-components/src/Badge/Badge.module.scss

```diff
@@ -14,7 +14,7 @@
 
 .intent-neutral {
   background: $crl-neutral-2;
-  color: $crl-base-text--dark;
+  color: $crl-base-text-dark;
```

Comment:
Did this because other token values are simply single-hyphen whereas the text values were the only token double-hyphened (in the BEM-flavored style).  I updated this (here and in tokens file) for consistency.